packages/pluggableWidgets/badge-native/package.json

```diff
@@ -27,7 +27,6 @@
   },
   "devDependencies": {
     "@mendix/pluggable-widgets-tools": ">=8.9.2",
-    "detox": "^18.16.0",
```

Comment:
a package-lock should indeed be in this PR

Reply:
I'm not able to find `package-lock.json` in `master`. Do you still want me to add it?

Reply:
Nope, package-lock should only lives in the root package.